elixir/support/app_identity/suite/generator.ex

```diff
@@ -229,29 +235,77 @@ defmodule AppIdentity.Suite.Generator do
     output
   end
 
-  defp normalize_padlock(output, %{"padlock" => padlock} = input, type, index) do
-    must_have!(type, padlock, index, "nonce", input: input, name: "padlock.nonce")
-    Map.put_new(output, :padlock, %{nonce: padlock["nonce"]})
+  defp normalize_padlock(output, %{"padlock" => %{"value" => _} = padlock} = input, type, index) do
```

Comment:
(nit) I know you didn’t change this, but an input called `output` is odd.

Reply:
This will be changed in v2 Elixir; I am not super happy with how some of the code is written here.